packages/nextjs/app/debug/debugContracts.tsx

```diff
@@ -1,15 +1,15 @@
+"use client";
+
 import { useEffect } from "react";
-import type { NextPage } from "next";
 import { useLocalStorage } from "usehooks-ts";
-import { MetaHeader } from "~~/components/MetaHeader";
 import { ContractUI } from "~~/components/scaffold-eth";
 import { ContractName } from "~~/utils/scaffold-eth/contract";
 import { getContractNames } from "~~/utils/scaffold-eth/contractNames";
 
 const selectedContractStorageKey = "scaffoldEth2.selectedContract";
 const contractNames = getContractNames();
 
-const Debug: NextPage = () => {
+export default function DebugContracts() {
```

Comment:
better to move file to `components/debug/DebugContracts.tsx`

Reply:
it is a component so export would be required ,right?

Reply:
yes, but default required only for pages